pages/common/gdu.md

```diff
@@ -0,0 +1,36 @@
+# gdu
+
+> Disk usage analyzer with console interface.
+> More information: <https://github.com/dundee/gdu>.
+
+- Show disk usage of the current directory:
+
+`gdu`
+
+- Show disk usage of given directory:
+
+`gdu {{path/to/directory}}`
+
+- Show disk usage of all mounted disks:
+
+`gdu --show-disks`
+
+- Show disk usage but ignore some paths:
+
+`gdu --ignore-dirs {{/sys,/proc}} /`
+
+- Ignore paths by regular pattern:
```

Comment:
Not sure if there's a difference between `regular pattern` and `regular expression`, but the latter is definitively more commmon.
```suggestion
- Ignore paths by regular expression:
```